applications/mne_analyze/plugins/datamanager/datamanager.cpp

```diff
@@ -144,11 +144,12 @@ QWidget *DataManager::getView()
 
 void DataManager::handleEvent(QSharedPointer<Event> e)
 {
-    switch (e->getType()) {
-    default:
-        qWarning() << "[DataManager::handleEvent] received an Event that is not handled by switch-cases";
-        break;
-    }
+    Q_UNUSED(e)
+//    switch (e->getType()) {
+//    default:
+//        qWarning() << "[DataManager::handleEvent] received an Event that is not handled by switch-cases";
+//        break;
+//    }
```

Comment:
Why was this commented out?

Reply:
After a long conversaition out in the RealWrold, it was simply a mistake on my side to comment it. I'll undo the changes.

Reply:
undone